scipy/stats/_continuous_distns.py

```diff
@@ -970,6 +975,73 @@ def _entropy(self):
 expon = expon_gen(a=0.0, name='expon')
 
 
+## Exponentially Modified Gaussian (exponential distribution
+##  convolved with a Gaussian)
+class expongauss_gen(rv_continuous):
+    """An exponentially modified Gaussian continuous random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for `expongauss` is::
+
+        expongauss.pdf(x, K) = 1/(2*K) exp(1/(2 * K**2)) exp(-x / K) * erfc(- (x - 1/K) / sqrt(2))
+           
+    where the shape parameter ``K`` > 0.
+
+    It can be thought of as the sum of a normally distributed random
+    value with mean ``loc`` and sigma ``scale`` and an exponentially
+    distributed random number with a pdf proportional to exp(-lambda * x)
+    where lambda = (``K`` * ``scale``)**(-1).
+
+    .. versionadded:: 0.16.0
+
+    %(example)s
+
+    """
+    def _argcheck(self, K):
+        return (K > 0)
```

Comment:
This is not needed, it's the default _argcheck, as inherited from rv_continuous.